---
layout: page
title: PH525x series - Biomedical Data Science
---

### Links and resources

- [The repository of the R markdown files for the pages below](http://github.com/genomicsclass/labs)
- [Links to the HarvardX class pages](pages/classes.html)
- [Online resources and books](pages/resources.html)
- [Finding more help for data analysis](pages/more_help.html)

### 1: Statistics and R for the Life Sciences

- [Getting started](pages/getting_started.html)
- [Setting up R and RStudio in Windows](https://github.com/genomicsclass/windows/blob/master/README.md)
- [R refresher](pages/R_refresher.html)
- [Installing packages from Github](pages/github.html)

---

- [Random variables](pages/random_variables.html)

---

- [Populations and samples](pages/populations_and_samples.html)
- [Central Limit Theorem and t-distribution](pages/clt_and_t-distribution.html)
- [Central Limit Theorem in practice](pages/clt_in_practice.html)
- [t-tests in practice](pages/t-tests_in_practice.html)
- [Confidence intervals](pages/confidence_intervals.html)
- [Association tests](pages/association_tests.html)
- [Power calculations](pages/power_calculations.html)
- [Monte Carlo methods](pages/monte_carlo.html)
- [Permutations tests](pages/permutation_tests.html)

---

- [Exploratory Data Analysis 1](pages/exploratory_data_analysis_1.html)
- [Exploratory Data Analysis 2](pages/exploratory_data_analysis_2.html)
- [Plots to avoid](pages/plots_to_avoid.html)
- [dplyr tutorial](pages/dplyr_tutorial.html)
- [Robust summaries](pages/robust_summaries.html)
- [Rank tests](pages/ranktest.html)

### 2: Introduction to Linear Models and Matrix Algebra

- [Introduction to Linear Models](pages/intro_using_regression.html)
- [Matrix Algebra Notation](pages/matrix_notation.html)
- [Matrix Algebra Operations](pages/matrix_operations.html)
- [Linear Models Expressed as Matrix Algebra](pages/matrix_algebra_examples.html)

---

- [Expressing design formula in R](pages/expressing_design_formula.html)
- [Linear models in practice](pages/linear_models_in_practice.html)
- [Standard errors](pages/standard_errors.html)
- [Interactions and contrasts](pages/interactions_and_contrasts.html)
- [Collinearity](pages/collinearity.html)
- [QR and regression](pages/qr_and_regression.html)
- [Going further](pages/linear_models_going_further.html)


### 3: Advanced Statistics for the Life Sciences

- [Introduction and example of high-throughput data](pages/intro_to_highthroughput_data.html)
- [Inference on high-throughput data](pages/inference_for_highthroughput.html)
- [EDA of high-throughput data](pages/eda_for_highthroughput.html)
- [Quick introduction to Bioconductor](pages/quick_Bioc_intro.html)

---
 
- [Mathematical definition of distance](pages/distance.html)
- [Introduction to the singular value decomposition (SVD)](pages/svd.html)
- [Dimension reduction with multidimensional scaling (MDS)](pages/mds.html)
- [Clustering and heatmaps](pages/clustering_and_heatmaps.html)
- [Introduction to Machine Learning](pages/machine_learning.html)
- [Crossvalidation](pages/crossvalidation.html)

---

- [Confounding](pages/confounding.html)
- [Adjusting for batch effects with linear models](pages/adjusting_with_linar_models.html)
- [Projections](pages/projections.html)
- [Rotations](pages/rotations.html)
- [Principal Component Analysis](pages/PCA.html)
- [Factor Analysis](pages/factor_analysis.html)
- [EDA with PCA](pages/eda_with_pca.html)
- [Adjusting for batch effects with factor analysis](pages/adjusting_with_factor_analysis.html)

### 4: Introduction to Bioconductor

- [Installing Bioconductor and finding help](pages/installing_Bioconductor_finding_help.html)
- [Annotating phenotypes and molecular function](pages/annoPhen.html)
- [The ExpressionSet Container](pages/eset.html)

---

- [IRanges and GRanges](pages/iranges_granges.html)
- [Operating on GRanges](pages/operateGRanges.html)

---

- [Cheat sheet for genomic annotation](pages/annoCheat.html)
- [Translating addresses between reference builds with liftOver](pages/anno4liftover.html)
- [Cheat sheet for GRanges and other Bioconductor objects](https://github.com/mikelove/bioc-refcard/blob/master/README.Rmd)

---

- [Importing NGS data with Bioconductor](pages/import_NGS.html)
- [NGS read counting](pages/read_counting.html)

---

- [Technical versus biological variability](pages/biological_versus_technical_var.html)
- [Statistical Inference with Bioconductor](pages/inference_with_bioc.html)
- [Using limma](pages/using_limma.html)
- [Gene Sets Analysis](pages/gene_set_analysis.html)
- [Gene Sets Analysis in R](pages/gene_set_analysis_in_R.html)


<!--

new ones:
- [Computing with chromosomes and variants](pages/chromComp.html)

old ones:
- [Reading microarray data](pages/reading_microarray_data.html)
- [Downloading data from GEO using GEOquery](pages/GEOquery.html)
- [Background](pages/background.html)
- [Normalization](pages/normalization.html)
- [SVA, ComBat, PCA and SVD](pages/svacombat.html)
- [Basic inference for microarray](pages/basic_inference_microarray.html)
- [EDA plots for microarray](pages/EDA_plots_for_microarray.html)
- [EDA plots for next generation sequencing](pages/EDA_plots_for_NGS.html)
- [Visualizing NGS data](pages/visualizing_NGS.html)
- [Mapping features to genes](pages/mapping_features.html)
- [Gene set analysis lecture](pages/gsa.html)
- [Gene set testing in R](pages/gene_set_testing.html)


-->

### 5: RNA-seq data analysis

- [Downloading and unzipping fastq files](https://github.com/genomicsclass/labs/blob/master/course5/fastq.md)
- [Genomic alignment with STAR](https://github.com/genomicsclass/labs/blob/master/course5/genome_align_STAR.md)
- [Transcriptome alignment with RSEM](https://github.com/genomicsclass/labs/blob/master/course5/trancsript_align_RSEM.md)

---

- [RNA-seq at the gene-level: EDA, DE and SVA](pages/rnaseq_gene_level.html)
- [Differential exon usage](pages/rnaseq_exon_usage.html)
- [Exploring plots of isoform-level expression with Cufflinks/cummeRbund](pages/rnaseq_isoform_cummerbund.html)

### 6: Variant Discovery and Genotyping

- [Genome variation](pages/SNP.html) from 2014

### 7: ChIP-seq data analysis

- [ChIP-seq](pages/ChIPseq.html) from 2014

### 8: DNA methylation data analysis

- [DNA Methylation Data Analysis](pages/methylation.html)
- [Reading 450K idat files with the minfi package](pages/minfi.html)
- [Interactive visualization of DNA methylation data analysis](pages/epiviz.html)
 
 ---
 
 - [Statistical Inference in the Analysis of DNA methylation Data](pages/inference_for_DNAmeth.html)

---

### Footnotes for all lectures

- [Footnotes](pages/footnotes.html)

### Acknowledgments

- [Course acknowledgments](pages/acknowledgments.html)

